### PR TITLE
Plugins Multisite - Update the plugin autoupdate tooltip visibility

### LINKED ISSUE #67541: Plugins: Autoupdate toggle popover info shouldn't hide together with the label
We are hiding the auto-update labels on Desktop and showing them only in Mobile. This includes the popover which should have been showing since it contains valuable information.

Image

Steps to reproduce:
- install `wordpress-seo` to any of your sites
- visit `/plugins/wordpress-seo`
- click "Manage sites"

### PR DESCRIPTION
**Pull request by WBerredo**
#### Proposed Changes

Update the plugin autoupdate tooltip visibility:
- Change the hiding mode of the label to `visibility: hidden` instead of `display: none`
- Update the tooltip to be always visible, even when the label is invisible
- Update the `z-index` to allow the tooltip to be shown on the modal

#### Testing Instructions

* Go to a multisite plugins page. Ex: `/plugins/woocommerce-subscriptions` PS: no site at the end of the URL
* Make sure you install or have it installed in at least one site
* Click on manage sites
* Check if the autoupdate option
  * is disabled
  * have a `i` icon on the side
  * you can see a explanation why it is disabled after clicking on the `i`

| Before  | After |
| ------------- | ------------- |
|Screen Shot 2022-09-19 at 10 20 04|Screen Shot 2022-09-19 at 10 20 24|

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

---

Fixes #67541

### CODE REVIEW

File: client/my-sites/plugins/plugin-action/plugin-action.jsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ class PluginAction extends Component {
 				ref={ this.disabledInfoLabelRef }
 				onClick={ this.handleAction }
 			>
-				{ this.props.label }
-				{ this.renderDisabledInfo() }
+				<span className="plugin-action__label-text">{ this.props.label }</span>
+				<span className="plugin-action__label-disabled-info">{ this.renderDisabledInfo() }</span>
```
Comment (epeicher) [inline client/my-sites/plugins/plugin-action/plugin-action.jsx:35]: Just checking, where is the class `plugin-action__label-disabled-info` used?

File: client/my-sites/plugins/plugin-action/style.scss [hunk 2]
```diff
@@ -52,7 +52,11 @@
 	cursor: pointer;
 
 	&.hide {
-		display: none;
+		visibility: hidden;
+
+		.plugin-action__label-text {
+			display: none;
+		}
 	}
```
Comment (epeicher) [inline client/my-sites/plugins/plugin-action/style.scss:61]: nit-pick: It seems that the `i` icon is not totally aligned by a couple of pixels 😅. 
CleanShot 2022-09-19 at 19 15 15@2x


Adding a `display: flex` solves that but I will leave it to you to decide as it maybe looks better being top-aligned
```suggestion
.plugin-action__label-disabled-info {
	display: flex;
}
```

**Review by epeicher — APPROVED** [review]
This works as advertised! I have left a couple of questions but the changes LGTM

File: client/my-sites/plugins/plugin-action/plugin-action.jsx (hunk 1 above)
Comment (WBerredo) [reply client/my-sites/plugins/plugin-action/plugin-action.jsx:35]: Now its being used to apply your suggestion: https://github.com/Automattic/wp-calypso/pull/67984/commits/29a1de4023ee3ad7043830c071d9b9bec3a0b319 :D

File: client/my-sites/plugins/plugin-action/style.scss (hunk 2 above)
Comment (WBerredo) [reply client/my-sites/plugins/plugin-action/style.scss:61]: Thanks for the suggestion, applied at https://github.com/Automattic/wp-calypso/pull/67984/commits/29a1de4023ee3ad7043830c071d9b9bec3a0b319

**Pull request merged**